genesyscloud/flow_loglevel/resource_genesyscloud_flow_loglevel_utils.go

```diff
@@ -0,0 +1,83 @@
+package flow_loglevel
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v125/platformclientv2"
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+)
+
+func generateFlowLogLevelResource(
+	flowId string,
+	flowLoglevel string,
+	resourceId string,
+) string {
+	return fmt.Sprintf(`resource "genesyscloud_flow_loglevel" "%s" {
+	  flow_id					= "%s"
+	  flow_log_level 			= "%s"
+	}`,
+		resourceId,
+		flowId,
+		flowLoglevel)
+}
+
+// getFlowLogLevelSettingsRequestFromResourceData maps data from schema ResourceData object to a platformclientv2.Flowloglevelrequest
+func getFlowLogLevelSettingsRequestFromResourceData(d *schema.ResourceData) platformclientv2.Flowloglevelrequest {
```

Comment:
This function isn't really doing a lot, It would be easier if you create the `Flowloglevelrequest` directly in the create and update.